### PR TITLE
Release 1.15.0

### PR DESCRIPTION
**Pull request by lmakarov**
## New software versions

- fin v1.101.0
- Docker Desktop v2.5.0.1 (49550)
- Docker v19.03.13 (boot2docker pinned at 19.03.5)
- Docker Compose v1.27.4
- VirtualBox v6.1.16
- Switched `cli` service to `docksal/cli:2.12-php7.3`
  - Dropped PHP 7.2 (EOL as of 30 Nov 2020)
  - Added Composer v2 (default) with ability to pin to Composer v1
  - Updated NodeJs to the latest LTS release v14.15.0
  - Version bumps for other console tools and runtimes
  - See https://github.com/docksal/service-cli/releases/tag/v2.12.0 for  more details.
- Switched `postgres` service to `postgres:10.14-alpine`
  - Drupal 9 requires PostgreSQL 10+

## New features

- macOS Big Sur support

## Changes and improvements

- Update for Elasticsearch 7 compatibility
- Adjusted approach to mounting ssh-agent socket in cli (#1391)
- Prevent PTY allocation for mysql when no TTY is available (#1394)
- Fixed `SSH_AUTH_SOCK_DIR` owner in cases when the volume directory has been already created by docker (root user)
- Fixed target repositories to match project create selection
- Increased precision in `ver_to_int` to 4 positions to correctly handle Docker Desktop's a.b.c.d (4 positions) version schema
- Fixed `docker_desktop_version` check on Windows for recent Docksal Desktop versions
- Fixed bind mounts on Docker Desktop for Mac/Windows v2.3.0.2+
- Changed the value for `DOCKASL_VOLUME=disabled` (`DOCKASL_VOLUME=disable` => `DOCKASL_VOLUME=disabled`)
- Removed Acquia drush commands (Acquia Cloud API v1 is EOL)
- Added support for branch name when using a custom git repo in `fin project create`

## Documentation

- Updated troubleshooting docs on conflicting ports (how to identify which process uses a port)
- Updated VPN troubleshooting docs
  - Mentioned that the VPN workaround is only possible with Docker Desktop for Mac/Windows (see #1397)
- Added docs for missing SSE4.2 CPU instruction set (#1414)
- Fixed Dockerfile example to use proper shell in "Extending Stock Images" docs
- Fixed ssh-key help message
- Added documentation for the Acquia CLI tool (#1387)
- Using entirely custom docker-compose configurations
- Refactored stack configuration docs
- Added Composer version docs in service/cli/settings
- Fixed internal linking in docs
- Wordpress multisite settings
- Updated project's README on GitHub

### CODE REVIEW

**Comment by shelane** [comment]
If we're bumping Docksal versions, the versions file needs one more update.

File: bin/fin [hunk 1]
```diff
@@ -3275,12 +3275,12 @@ project_create ()
 		2)
 			target_cms="Drupal 9 (BLT Version)"
 			target_host_name_search_string="drupal9.docksal"
-			target_repo="$URL_REPO_DRUPAL9COMPOSER"
+			target_repo="$URL_REPO_DRUPAL9BLT"
```
Comment (danshumaker) [inline bin/fin:3280]: Thank you @shelane for this. I'm using fin 1.99 and just created a #2 project and didn't get BLT installed... I was like "WUT!!" .. then looked at the source code and was about to submit a patch doing what you just did above!  #beat-me-2-it  .. Glad to see it's fixed in the next version!

**Review by achekulaev — APPROVED** [review]

**Review by sean-e-dietrich — APPROVED** [review]

File: bin/fin [hunk 2]
```diff
@@ -136,20 +136,20 @@ wsl_path ()
...
 REQUIREMENTS_WINPTY_CYGWIN='2.8.0'
-REQUIREMENTS_DOCKER_DESKTOP='2.1.0.5'  # This is the last version before networking regressions were introduced (#1268)
+REQUIREMENTS_DOCKER_DESKTOP='2.5.0.1'  # This is the last version before networking regressions were introduced (#1268)
 
 # VirtualBox download URLs
 # Remember to update REQUIREMENTS_VBOX above
-URL_VBOX_MAC="https://download.virtualbox.org/virtualbox/6.1.10/VirtualBox-6.1.10-138449-OSX.dmg"
-URL_VBOX_WIN="https://download.virtualbox.org/virtualbox/6.1.10/VirtualBox-6.1.10-138449-Win.exe"
+URL_VBOX_MAC="https://download.virtualbox.org/virtualbox/6.1.16/VirtualBox-6.1.16-140961-OSX.dmg"
+URL_VBOX_WIN="https://download.virtualbox.org/virtualbox/6.1.10/VirtualBox-6.1.16-140961-Win.exe"
```
Comment (lpeabody) [inline bin/fin:152]: 6.1.10 part of the slug should be 6.1.16:

https://download.virtualbox.org/virtualbox/6.1.16/VirtualBox-6.1.16-140961-Win.exe

**Review by lpeabody — CHANGES_REQUESTED** [review]
Just need to update the single URL variable referencing the download location for Virtual Box for Windows.

File: bin/fin (hunk 2 above)
Comment (lmakarov) [reply bin/fin]: Good catch! Thanks 👍

**Pull request merged**